packages/log/src/DefaultLogger.ts

```diff
@@ -45,15 +45,17 @@ class DefaultLogger extends Logger {
     this.modules = {};
   }
 
-  module(name) {
+  modules: Record<string, Logger>;
+
+  module(name: string): Logger {
     if (!this.modules[name]) {
       this.modules[name] = new Logger(name, this.level);
     }
 
     return this.modules[name];
   }
 
-  setLogLevel(level) {
+  setLogLevel(level: number | undefined): void {
```

Comment:
I think this should always be a `number` - why allow `undefined` here?

Reply:
The constructor can be called with an undefined level, and setLogLevel is called there. But now that I'm "manually" defaulting the log functions to silent, I don't think I have to call setLogLevel anymore if level is undefined?